API/common/utils.py

```diff
@@ -210,10 +212,60 @@ def relpath(path, start):
     return os.path.relpath(path, start)
 
 
+def rmtree(path):
+    '''
+    Remove directory
+    '''
+    if exists(path):
+        shutil.rmtree(path)
+
+
+def get_section_sizes_from_map(mapfile):
+    '''
+    Returns the sizes of the main sections.
+    '''
+
+    archives = ['libhttpparser.a',
+            'libiotjs.a',
+            'libjerry-core.a',
+            'libjerry-ext.a',
+            'libjerry-port-default.a',
+            'libjerry-port-default-minimal.a',
+            'libtuv.a']
+
+    print("Loading map file '%s'" % mapfile)
```

Comment:
Should we keep this line?

Reply:
IMHO it is a useful information, but I can remove it if it does not fit.